extensions/prometheus/PrometheusExposerWrapper.cpp

```diff
@@ -20,9 +20,27 @@
 
 namespace org::apache::nifi::minifi::extensions::prometheus {
 
-PrometheusExposerWrapper::PrometheusExposerWrapper(uint32_t port)
-    : exposer_(std::to_string(port)) {
-  logger_->log_info("Started Prometheus metrics publisher on port %" PRIu32, port);
+PrometheusExposerWrapper::PrometheusExposerWrapper(const PrometheusExposerConfig& config)
+    : exposer_(parseExposerConfig(config)) {
+  logger_->log_info("Started Prometheus metrics publisher on port %" PRIu32, config.port);
```

Comment:
It could be useful to add "with TLS enabled" to this log message if the config contains a certificate.

Reply:
Updated in 45f72a651e07636f89cb177f8536a78cf3fe67d3